diff --git a/node_modules/@types/jscodeshift/src/collections/Node.d.ts b/node_modules/@types/jscodeshift/src/collections/Node.d.ts
index d1e74f3..7eb7496 100644
--- a/node_modules/@types/jscodeshift/src/collections/Node.d.ts
+++ b/node_modules/@types/jscodeshift/src/collections/Node.d.ts
@@ -8,7 +8,7 @@ export interface TraversalMethods {
     /**
      * Find nodes of a specific type within the nodes of this collection.
      */
-    find<T>(type: recast.Type<T>, filter?: ((value: any) => boolean) | object): Collection.Collection<T>;
+    find<T>(type: recast.Type<T>, filter?: ((value: T) => boolean) | object): Collection.Collection<T>;
 
     /**
      * Returns a collection containing the paths that create the scope of the
